.github/workflows/build-and-publish.yml

```
@@ -60,10 +87,27 @@ jobs:
         updateOnlyUnreleased: true
 
     - name: VirusTotal Scan
+      if: secrets.VT_API_KEY != ''
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)